libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartStore/SFSmartStorePlugin.m

```diff
@@ -351,8 +343,7 @@ - (void)pgReIndexSoup:(CDVInvokedUrlCommand *)command
 - (void)pgShowInspector:(CDVInvokedUrlCommand *)command
 {
     [self runCommand:^(NSDictionary* argsDict) {
-        BOOL isGlobal = [self isGlobal:argsDict];
-        SFSmartStoreInspectorViewController* inspector = isGlobal ? self.globalInspector : self.inspector;
+        SFSmartStoreInspectorViewController* inspector = [self inspector:command];
         [inspector present:self.viewController];
```

Comment:
One more request: can we do away with this pattern as well, and the `dismiss` pattern in `SFSmartStoreInspectorViewController`?  They're both substituting for standard view controller presentation patterns:

`[self.viewController presentViewController:inspector animated:NO completion:nil];`

and
```obj-c
- (void) backButtonClicked
{
    [self.presentingViewController dismissViewControllerAnimated:NO completion:NULL];
}
```